README.md

```diff
@@ -84,7 +84,7 @@ server.start((err) => {
 `ipfsd-ctl` can spawn `disposable` and `non-disposable` daemons.
 
 - `disposable`- Creates on a temporary repo which will be optionally initialized and started (the default), as well cleaned up on process exit. Great for tests.
-- `non-disposable` - Requires the user to initialize and start the node, as well as stop and cleanup after wards. Additionally, a non-disposable will allow you to pass a custom repo using the `repoPath` option, if the `repoPath` is not defined, it will use the default repo for the node type (`$HOME/.ipfs` or `$HOME/.jsipfs`). The `repoPath` parameter is ignored for disposable nodes, as there is a risk of deleting a live repo.
+- `non-disposable` - Non disposable daemons will by default attach to any nodes running on the default or the supplied repo. Requires the user to initialize and start the node, as well as stop and cleanup after wards. Additionally, a non-disposable will allow you to pass a custom repo using the `repoPath` option, if the `repoPath` is not defined, it will use the default repo for the node type (`$HOME/.ipfs` or `$HOME/.jsipfs`). The `repoPath` parameter is ignored for disposable nodes, as there is a risk of deleting a live repo.
```

Comment:
"Requires the user to initialize and start the node, as well as stop and cleanup after wards"

This is incorrect. It should spawn if there is not already one.

Reply:
With the current changes this PR isn't a breaking change. Because the previous behavior was basically a bug and now we can just pass start: true and it starts. 
I would like this to be explict, only start if the user tells it to start either by option or manually calling the start method.

I can re-work this a bit to explain this better. 

To auto start by default we should open a new issue and discuss the breaking change.

Reply:
When it is `disposable` it autostarts, it is confusing that `non-disposable` doesn't autostart.

Reply:
I have two points to keep bikeshedding this, sorry :/
- it's out of this PR scope which primarily fixes a bug according to the current behaviour
- making it auto start is a breaking change lots of repos that rely on the current behaviour will need to change code/tests to explicitly pass `init: false, start: false`

I would really prefer to talk/change this on a different issue/pr.

Reply:
Valid points